Component/Badge/badge.css

```diff
@@ -0,0 +1,75 @@
+@import url("../../Utility CSS/navbar.css");
+
+.box {
+  height: 14rem;
+  width: 120rem;
+  justify-content: space-around;
+  align-items: center;
+  padding: 2rem;
+}
+
+figure {

```

Comment:
avoid using direct tag selector to apply styles

Reply:
ok have to fix this for all the components then will fix it at the end